lib/ae_mdw/db/contract.ex

```diff
@@ -246,7 +246,7 @@ defmodule AeMdw.Db.Contract do
 
   @spec call_fun_arg_res(pubkey(), integer()) :: Contract.fun_arg_res()
   def call_fun_arg_res(contract_pk, call_txi) do
-    create_txi = Origin.tx_index!({:contract, contract_pk})
+    create_txi = Origin.tx_index!(State.new(), {:contract, contract_pk})
```

Comment:
This PR is preparation for accessing in-memory state, right? I understand the idea might be incremental PRs but as it deals already with lots of function signatures, would you mind using already the `State` from `StatePlug` for every read operation? It would help me with the reviews of the next PRs without changes regarding the use of `StatePlug`.

Reply:
Done